.sbtopts

```diff
@@ -0,0 +1,16 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Having a banner in this file is actually humorous.

Reply:
I don't disagree. But this squashes any chance of people bringing up license issues. With all the project Apache oversees, I think it's too time consuming for them to discuss all the edge cases and eal with grey areas. So I think their mantra is along the lines of "if it can have have a license in it, then it should", even when it's kindof ridiculous like this.